sys-boot/raspberrypi-firmware/raspberrypi-firmware-9999.ebuild

```diff
@@ -42,8 +43,8 @@ pkg_preinst() {
 
 src_install() {
 	insinto /boot
-	cd boot
-	doins bootcode.bin COPYING.linux fixup*.dat LICENCE.broadcom start*elf
+	cd boot || die
+	doins bootcode.bin fixup*.dat start*elf
 	doins *.dtb
 	doins -r overlays
```

Comment:
Do we really want overlays? If a user compiles a kernel, this installs the overlays.